_episodes/l1-02-branching-merging.md

```diff
@@ -0,0 +1,578 @@
+  (use "git merge --abort" to abort the merge)
+
+Unmerged paths:
+  (use "git add <file>..." to mark resolution)
+	both modified:   ingredients.md
+
+no changes added to commit (use "git add" and/or "git commit -a")
+~~~
+
+This suggests how we can get out of this state. If we want to give up on this
+merge and try it again later then we can use `git merge --abort.`. This will
+return the repository to its pre-merge state. We will likely have to deal with
+the conflict at some point though so may as well do it now. Fortunately we don't
+need any new commands. We just need to edit the conflicted file into the state
+we would like to keep, then add and commit as usual.
+
+Let's look at ingredients.md to understand the conflict:
+~~~
+* 2 avocados
+* 1 lime
+<<<<<< HEAD
+* 1 tsp salt
+=======
+* 3 tsp salt
+>>>>>> experiment
+* 1/2 onion
+* 1 tbsp coriander
+~~~
+
+Git has changed this file for us and added some lines which highlight the
+location of the conflict. This may be confusing at first glance (a good editor
+may add some highlighting which can help), but you are essentially being asked
+to choose between the two versions presented. The tags `<<<<<<< HEAD` and
+`>>>>>>> experiment` are used to indicate which branch each version came from
+(HEAD here corresponds to `main` as that is our checked out branch). 
+
+The conflict makes sense, we can either have 1 tsp of salt or 3. There is no way
+for Git to know which it should be so it has to ask you. Let's resolve it by
+choosing the version from the main branch. Edit `ingredients.md` so it looks
+like:
+~~~
+* 2 avocados
+* 1 lime
+* 1 tsp salt
+* 1/2 onion
+* 1 tbsp coriander
+~~~
+
+now stage, commit and check the result:
+~~~
+$ git add ingredients.md
+$ git commit -m "Merged experiment into main"
+$ git graph
+~~~
+{: .commands}
+~~~
+*   e361d2b (HEAD -> main) Merged experiment into main
+|\  
+| * d5fb141 (experiment) Added salt to balance coriander
+* | 7477632 reduce salt
+* |   567307e Merge branch 'experiment'
+|\ \  
+| |/  
+| * 9a4b298 Reduced the amount of coriander
+* |   40070a5 Merge branch 'experiment'
+|\ \  
+| |/  
+| * 96fe069 try with some coriander
+* | d4ca89f Corrected typo in ingredients.md
+|/  
+* ddef60e Revert "Added instruction to enjoy"
+* 8bfd0ff Added 1/2 onion to ingredients
+* 2bf7ece Added instruction to enjoy
+* ae3255a Adding ingredients and instructions
+~~~
+{: .output}
+
+![Git collaborative]({{ site.baseurl }}/fig/branch9.png
+"Repository with third merge"){:class="img-responsive"}
+
+## Multiple branches in remotes
+
+The same way you might have different branches in your local repository, you could
+manage different branches in your remote - the same branches or different ones.
+
+As a reminder, remote and local repositories are not automatically syncronized, but
+rather it is a manual process done via `git pull` and `git push` commands. This
+syncronisation needs to be done **branch by branch** with all of those you want to keep
+in sync.
```

Comment:
```suggestion
As a reminder, remote and local repositories are not automatically synchronised, but
rather it is a manual process done via `git pull` and `git push` commands. This
synchronisation needs to be done **branch by branch** with all of those you want to keep
in sync.
```

Reply:
I've used British spelling here, but there are a bunch of words ending in -ize elsewhere. I'm happy to fix up

Reply:
Don't worry, I'll check those